src/python/bot/fuzzers/libFuzzer/strategy_selection.py

```diff
@@ -0,0 +1,105 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Generate strategy pool for launcher based on strategy probabilities.
+
+Decides the set strategies to be considered by the launcher. Note
+that because of compatability issues, the exact set of strategies
+generated here may be modified in the launcher before being launched."""
+
+from bot.fuzzers import engine_common
+from bot.fuzzers import strategy
+
+# Probability of using `-max_len` option. Not applicable if already declared in
+# .options file.
+RANDOM_MAX_LENGTH_PROBABILITY = 0.15
+
+# Probability of doing ML RNN mutations on the corpus in this run.
+CORPUS_MUTATION_ML_RNN_PROBABILITY = 0.50
+
+# Probability of doing radamsa mutations on the corpus in this run.
+CORPUS_MUTATION_RADAMSA_PROBABILITY = 0.15
+
+# Number of radamsa mutations.
+RADAMSA_MUTATIONS = 2000
+
+# Maximum number of seconds to run radamsa for.
+RADAMSA_TIMEOUT = 3
+
+# Probability of recommended dictionary usage.
+RECOMMENDED_DICTIONARY_PROBABILITY = 0.10
+
+# Probability of using `-use_value_profile=1` option.
+VALUE_PROFILE_PROBABILITY = 0.33
+
+FORK_PROBABILITY = 0.50
+
+MUTATOR_PLUGIN_PROBABILITY = 0.50
+
+
+def choose_generator():
+  """Return whether to use radamsa, ml rnn, or no generator."""
+
+  radamsa_p = engine_common.get_strategy_probability(
```

Comment:
maybe `_prob` or `_probability` instead of instead of `_p`?
We tend not to abbreviate things in naming in CF.